src/composables/transferSendHandler.ts

```diff
@@ -0,0 +1,49 @@
+import { useRoute } from 'vue-router';
+
+import { TransferFormModel, WalletRouteMeta } from '@/types';
+import {
+  IS_EXTENSION,
+  MODAL_TRANSFER_SEND,
+  TRANSFER_SEND_DATA_LOCAL_STORAGE_KEY,
+} from '@/constants';
+import { useModals } from '@/composables';
+import { getLocalStorageItem, removeLocalStorageItem, setLocalStorageItem } from '@/utils';
+
+export function useTransferSendHandler() {
```

Comment:
Can you please provide some comment about the use case / reasoning of having it.